src/image.rs

```diff
@@ -62,9 +62,15 @@ impl<'docker> Image<'docker> {
     /// Deletes an image
     ///
     /// [Api Reference](https://docs.docker.com/engine/api/v1.41/#operation/ImagePrune)
-    pub async fn delete(&self) -> Result<Vec<Status>> {
+    pub async fn delete(&self, delete_options: &DeleteOptions) -> Result<Vec<Status>> {
```

Comment:
From a library compatibility perspective, it seems like the `delete` API should be unchanged, and a new API should be added for deleting with options.